databases/week2/More queries.sql

```diff
@@ -0,0 +1,31 @@
+SET NAMES utf8mb4;
+
+-- Get all tasks assigned to users whose email ends in @spotify.com --
+SELECT t.title
+FROM user_task ut 
+INNER JOIN task t ON ut.task_id=t.id 
+INNER JOIN (
+    SELECT u.id, u.email
+    FROM user u
+    WHERE email LIKE '%@spotify.com'
+) u ON ut.user_id=u.id;
+
+
+-- Get all taks for 'Donald Duck' where status 'not started' -- 
+SELECT DISTINCT user.name, task.id, task.title, task.description, status.name FROM user
+INNER JOIN user_task ON user.id = user_task.task_id
+INNER JOIN task ON task.id = user_task.task_id
+INNER JOIN status ON task.status_id = status_id
+WHERE user.name LIKE '%Donald Duck%' AND status.name = 'Not started';
+
+
+-- Get all task for 'Maryrose Meadows' that were created in september --
+SELECT * FROM user JOIN user_task ON user.id = user_task.user_id JOIN task ON user_task.task_id = task.id WHERE user.name = 'Aubry Polak' AND MONTH(created) = 05;
+
```

Comment:
Who is `Aubry Polak`?
`'Maryrose Meadows' `

You need to run your query before you submit it.